### DISCUSSION TITLE
Angular 9: Module not found: Error: Can't resolve '@google/model-viewer'

### DISCUSSION THREAD

**Discussion by takarka** [google/model-viewer#2209] [Q&A] [answered]
### Description

I can not use <model-viewer> web component in Angular 9; I did everything that shows them [there ](https://vaadin.com/learn/tutorials/using-web-components-in-angular)

**Comment by elalish** [comment]
We had a similar report in #969. Could you remix our [Glitch](https://model-viewer.glitch.me/) to demonstrate the trouble, or point us to some other link that we can debug?

**Comment by lupohirp** [comment]
can you import this? 

import '@google/model-viewer/dist/model-viewer';

**Comment by SaintJacques** [comment]
import "@google/model-viewer"

**Comment by anders8** [comment, marked as answer]
As of Angular 14, both of these will import:

- import "@google/model-viewer"
- import '@google/model-viewer/dist/model-viewer';

however, the second one (import '@google/model-viewer/dist/model-viewer';) will then fail with an error message:
node_modules/@google/model-viewer/dist/model-viewer.d.ts:897:27 - error TS2749: 'MeshoptDecoder' refers to a value, but is being used as a type here.

It's working for me with import "@google/model-viewer" **after** I also installed "@types/webxr" from npm

**Comment by bguiop004** [comment]


**Reply by elalish** [reply]
Do you think that's enough information for us to give you a helpful answer?
